docs/trex_known_issues.md

```diff
@@ -29,3 +29,14 @@ For information about what is new or has changed in each release, see the [Relea
 
 When an extension needs full data access and the user does not have full data permission on the workbook, Tableau currently allows the extension to run. However, Tableau will throw a console error when the extension calls `getUnderlyingData()` method. See [Handle full data access and permission errors]({{ site.baseurl }}/docs/trex_getdata.html#handle-full-data-access-and-permission-errors).
 
+
+### Tableau Extensions API library version 1.0.0 
+
+If you are debugging your extension, you might see this message in the Debugging Tools console window if you are using the released version of the 1.0.0 library (for example, `tableau-extensions-1.0.0.js`) with a version of Tableau that is newer than Tableau 2018.2.
+
+```
+This is an alpha version of the Extensions API. Please upgrade to an official release.
+
+```
+
+You can safely ignore this warning. However, if you are using the 1.0.0 library, be sure to upgrade to the latest version, or use or `tableau-extensions-1.latest.js`, so that you always pick up the most recent update of that version of the library when it becomes available.
```

Comment:
this seems to suggest we have some sort of CDN where people can point to extensions-1.latest.js, do we have that yet? I remember sean doing something for that for hackathon

Reply:
@jz-huang - no, we still don't have a CDN yet. It seems like we should be now.  Thanks for pointing this out. The way it is written does make it sound like they don't need to do anything if they use "latest", so I'll remove this. Also, there is a typo here. "or use or" !

Reply:
We don't have a 'release-able' cdn. We have a proof of concept.